mautic/mautic
# Minor fix to form submission

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Just minor to code 
 
Steps to reproduce: 
1. Create focus with form
2. See source code of preview and start of tag `<form`
3. You should see something like `mauticform<?php echo $formName ?>` on input hidden  line

Steps to test:
1. Repeat all steps
2: See source code and see correct name of hidden input (mauticform_formalias_focus_id)
